python/sglang/check_env.py

```diff
@@ -63,13 +76,22 @@ def get_cuda_info():
     """
     Get CUDA-related information if available.
     """
-    cuda_info = {"CUDA available": torch.cuda.is_available()}
+    if is_rocm:
```

Comment:
Make cuda as the first branch.

Reply:
the is_cuda returns true even on HIP :-)